### ISSUE TITLE
CREATE CURD operations of STACK

### ISSUE THREAD

**Issue by QMiNt** [QMiNt/Lets-Hack-october#3]

**Comment by AcidicArmadillo** [comment]
Hello, is this issue for writing the programs for Create, Read, Update and Delete operations of Stack data? If so, I would like to work on this issue.

**Comment by QMiNt** [comment]
Yes it is ! If you want to contribute any other code raise a issue.

**Comment by AcidicArmadillo** [comment]
Understood, Thanks!
